### DISCUSSION TITLE
AuthJs not connecting on digitalOcean

### DISCUSSION THREAD

**Discussion by chrishj59** [nextauthjs/next-auth#11399] [Help]
I am using AuthJs v5
I am getting an 0auth error when connect via the the sign button only when using the app deployed to DigitalOcean droplet. From local all works fine. 
I have AUTH_URL, AUTH_SECRET, and AUTH_TRUST=false in .env.local
How do I get it to work on production?
Thanks

**Comment by chrishj59** [comment]
Issue was with GitHub Auth0 App settings

**Reply by ivanhulakdev** [reply]
> Issue was with GitHub Auth0 App settings

Did you solve it, I have the same issue? I will appreciate any help, thanks in advance.
